akka-actor/src/main/scala/akka/util/Reflect.scala

```diff
@@ -154,4 +154,25 @@ private[akka] object Reflect {
       }
     }
   }
+
+  /**
+   * INTERNAL API
+   */
+  private[akka] def findClassLoader(): ClassLoader = {
+    def findCaller(get: Int ⇒ Class[_]): ClassLoader =
+      Iterator.from(2 /*is the magic number, promise*/ ).map(get) dropWhile { c ⇒
```

Comment:
then we must debug and verify that the behavior is the same (got a feeling that there are no tests for this, but I can be wrong)